### PR TITLE
Trust certificate

### PR DESCRIPTION
**Pull request by FredericBGA**
The pending PR for planemo is coming

### CODE REVIEW

**Comment by FredericBGA** [comment]
https://github.com/galaxyproject/planemo/pull/1010

File: lib/galaxy/tool_util/verify/interactor.py [hunk 1]
```diff
@@ -106,6 +106,15 @@ def __init__(self, **kwds):
         if kwds.get('user_api_key_is_admin_key', False):
             self.master_api_key = self.api_key
         self.keep_outputs_dir = kwds["keep_outputs_dir"]
+        self.verify = True
+        if kwds.get('trust', True):
```
Comment (jmchilton) [inline lib/galaxy/tool_util/verify/interactor.py:110]: Should the default here be False?

Comment (FredericBGA) [reply lib/galaxy/tool_util/verify/interactor.py:110]: Yes!
I will do that. I'm going to resend a PR without the commit related to my previous branch...

**Comment by FredericBGA** [comment]
I'm going to resend a PR without the commit related to my previous branch...

File: lib/galaxy/tool_util/verify/interactor.py (hunk 1 above)
Comment (FredericBGA) [reply lib/galaxy/tool_util/verify/interactor.py:110]: > 
> 
> Should the default here be False?

No. The default behavior of Request is to check the certificate (so this is the default of the Planemo test command), so `verify=True`. When you decide to trust the certificate with `--trust` then Request will use `verify=False`

Comment (FredericBGA) [reply lib/galaxy/tool_util/verify/interactor.py:110]: I've documented the option for planemo like this:

> @click.option(
>     "--trust",
>     is_flag=True,
>     help="When set, Requests will trust SSL certificates for HTTPS requests. "
>          "It sets verify=False in requests call",
>     default=False,
> )

Comment (nsoranzo) [reply lib/galaxy/tool_util/verify/interactor.py:110]: I'd suggest to replace the above 2 lines with:

```python
        self.verify = not kwds.get('trust', False)
        if not self.verify:
```

and remove line 117.

**Pull request closed without merging**